docs/developer/tutorials/snmp/how-to.md

```diff
@@ -249,6 +249,13 @@ For example:
 mib2dev.py --mib-module=<MIB> --start-oid=1.3.6.1.4.1.674.10892.1.400.20 --stop-oid=1.3.6.1.4.1.674.10892.1.600.12 > /path/to/mytable.snmprec
 ```
 
+The following command will generate 4 rows for the `IF-MIB:ifTable (1.3.6.1.2.1.2.2)`:
+
+```bash
+mib2dev.py --mib-module=IF-MIB --start-oid=1.3.6.1.2.1.2.2 --stop-oid=1.3.6.1.2.1.2.3 --table-size=4 > /path/to/mytable.snmprec
```

Comment:
I just tried, and it does not seem to work...

```
# Starting table IF-MIB::ifTable (1.3.6.1.2.1.2.2)
*** Inconsistent value: Display format eval failure: b'oxen quaintly zombies zombies acted': invalid literal for int() with base 16: 'oxen quaintly zombies zombies acted'caused by <class 'ValueError'>: invalid literal for int() with base 16: 'oxen quaintly zombies zombies acted'
```

Reply:
Honestly I copy pasted from snmpsim docs without trying. Checking it now.

Reply:
I could eventually test it, and seems there's a bug for `IF-MIB::ifPhysAddress` specifically in `mib2dev`. This field contains an hexstring that represents the MAC address of an interface, and `mib2dev` uses a random string. Fixed it by providing a valid MAC address (`00000000`)

Reply:
Updated docs to add a workaround for this error